lib/voxpupuli/acceptance/spec_helper_acceptance.rb

```diff
@@ -80,7 +80,7 @@ def configure_beaker(modules: :metadata, &block)
   c.add_setting :suite_hiera, default: true
   c.add_setting :suite_hiera_data_dir, default: File.join('spec', 'acceptance', 'hieradata')
   c.add_setting :suite_hiera_hierachy, default: [
-    'fqdn/%{fqdn}.yaml',
+    'fqdn/%{networking.fqdn}.yaml',
     'os/%{os.family}/%{os.release.major}.yaml',
     'os/%{os.family}.yaml',
```

Comment:
Looking at https://www.puppet.com/docs/puppet/7/hiera_config_yaml_5.html#config_syntax I see `facts.$fact` is used. I wonder if that's also supported with Hiera 3, which we're currently using.